nixos/modules/testing/test-instrumentation.nix

```diff
@@ -49,9 +56,49 @@ in
 
 {
 
+  options.testing = {
+
+    initrdBackdoor = lib.mkEnableOption (lib.mdDoc ''
```

Comment:
Can this be used to test soft-reboots? It would be cool if the description would also mention when to use this feature.

Reply:
@Mic92 I'm not sure what this would have to do with soft reboots?

I'll update the description

Reply:
Yes. Please. Neither the commit message nor the module description explain why we need this option.

Reply:
Doc added, resolving.